test/conformance/README.md

```diff
@@ -0,0 +1,81 @@
+# Protobom Conformance Testing
+
+This directory contains the protobom conformance test framework. It is still
+under construction but it already runs along the other project tests.
+
+## Conformance Testing Overview
+
+The main goal of the protobom conformance testing framework is to ensure that
+each protobom serialization or unserialization result in equivalent documents
+every time.
+
+SBOM diffing is a complex problem. SBOMs cannot be compared as simple strings or
+comparing data structs. Verifying that documents are equivalent needs more precise,
+semantic comparison. 
+
+The approach the protrobom conformance tests take is to ask the same group of
+questions to a generated and to its equivalente golden sample and contrast the
```

Comment:
Typo in this line?

> The approach the protrobom conformance tests take is to ask the same group of
questions to a generated [SBOM?]

Reply:
`equivalente` -> `equivalent`

Reply:
fixed :+1: